db/re/item_db_equip.yml

```diff
@@ -116389,6 +116603,15 @@ Body:
     MagicAttack: 175
     Range: 1
     Slots: 2
+    Jobs:
```

Comment:
Looks wrong. Should be Summoner only?

Reply:
The item is https://www.divine-pride.net/database/item/26151

Reply:
![grafik](https://user-images.githubusercontent.com/3517879/147110224-a2cbe1e2-419e-4658-be6e-b43560e5b8b9.png)

Thank you github!